src/styles/utilities.styl

```diff
@@ -27,11 +27,16 @@
     .u-w-100
         width 100%
 
-    .u-click-xl::after
-      width 130%
-      height 130%
-      top -15%
-      left -15%
+    // TODO Move utility to cozy-ui
+    .u-click-200
```

Comment:
I'm not sure u-click-200 is the good name to give to this class. If we want to increase the clickable area, it will not be representative anymore.

Reply:
Replaced by `u-expanded-click-area`